src/react/account/AccountPoliciesList.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button } from '@scality/core-ui/dist/next';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import { formatShortDate } from '../utils';
+import { useIAMClient } from '../IAMProvider';
+import CopyARNButton from '../ui-elements/CopyARNButton';
+import { Tooltip } from '@scality/core-ui';
+import { SpacedBox } from '@scality/core-ui/dist/components/spacedbox/SpacedBox';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useMutation, useQuery } from 'react-query';
+import { queryClient } from '../App';
+import DeleteConfirmation from '../ui-elements/DeleteConfirmation';
+import { Banner } from '@scality/core-ui';
+import { getPoliciesQuery } from '../queries';
+import { Icon } from '../ui-elements/Help';
+import AwsPaginatedResourceTable from './AwsPaginatedResourceTable';
+import IAMClient from '../../js/IAMClient';
+
+const RenderEditButton = ({ policyName }: { policyName: string }) => {
+  const history = useHistory();
+  const storageAccountOwnerPolicy = 'storage-account-owner-policy';
+  const storageManagerPolicy = 'storage-manager-policy';
+  const disableCondition = [storageAccountOwnerPolicy, storageManagerPolicy].includes(policyName);
+  return (
+    <SpacedBox ml={12}>
+      <Button
+        style={{ height: spacing.sp24 }}
+        disabled={ disableCondition }
+        variant='secondary'
+        label='Edit'
+        icon={<i className='fa fa-pen'></i>}
+        onClick={() => history.push(`users/${policyName}/update-policy`)}
+        tooltip={{ overlay: disableCondition ? 'You cannot edit this predefined Scality Policy' : '', placement: 'right' }}
+      />
+    </SpacedBox>);
+};
+
+const RenderAttachButton = () => {
+  const history = useHistory();
+  return (
+    <SpacedBox ml={12}>
+      <Button
+        style={{ height: spacing.sp24 }}
+        variant='secondary'
+        label='Attach'
+        icon={<i className='fas fa-link'></i>}
+        onClick={() => history.push('attach-user-policy')}
+      />
+    </SpacedBox>);
+};
+
+const renderActionButtons = (rowValues, accountName) => {
+  const { arn, policyName } = rowValues;
+  return (
+    <div style={{ display: 'flex' }}>
```

Comment:
there is a box component from core-ui 
```suggestion
    <Box display='flex' }}>
```